src/vivarium/framework/components/parser.py

```diff
@@ -180,85 +152,84 @@ def _process_level(level, prefix):
 
         component_list = []
         for name, child in level.items():
-            component_list.extend(_process_level(child, prefix + [name]))
+            component_list.extend(self._process_level(child, prefix + [name]))
 
         return component_list
 
-    return _process_level(component_config, [])
+    def prep_components(
+        self, component_list: Union[List[str], Tuple[str]]
+    ) -> List[Tuple[str, Tuple]]:
+        """Transform component description strings into tuples of component paths
+        and required arguments.
 
+        Parameters
+        ----------
+        component_list
+            The component descriptions to transform.
 
-def prep_components(component_list: Union[List[str], Tuple[str]]) -> List[Tuple[str, Tuple]]:
-    """Transform component description strings into tuples of component paths
-    and required arguments.
+        Returns
+        -------
+        List[Tuple[str, Tuple]]
+            List of component/argument tuples.
+        """
+        components = []
+        for c in component_list:
+            path, args_plus = c.split("(")
+            cleaned_args = self.clean_args(args_plus[:-1].split(","), path)
```

Comment:
Does `self.clean_args()` work when clean_args is static?

Reply:
Yes, you can call a static method either off the class itself or off an instance of it:  i.e. `self.static_method` or `Class.static_method`